database/redis/contact.go

```diff
@@ -66,22 +66,44 @@ func (connector *DbConnector) GetAllContacts() ([]*moira.ContactData, error) {
 	switch c := client.(type) {
 	case *redis.ClusterClient:
 		err := c.ForEachMaster(connector.context, func(ctx context.Context, shard *redis.Client) error {
-			iter := shard.Scan(ctx, 0, contactKey("*"), 0).Iterator()
-			for iter.Next(ctx) {
-				keys = append(keys, iter.Val())
+			var cursor uint64
+			var err error
+			var keysResult []string
+			for {
+				keysResult, cursor, err = shard.Scan(connector.context, cursor, contactKey("*"), 10000).Result()
```

Comment:
Maybe create function level const for 10000?

Reply:
Please double-check if there should be `ctx` instead of `connector.context` on that line

Reply:
Done